package/com.unity.formats.usd/Documentation~/USD Export.md

```diff
@@ -0,0 +1,77 @@
+# Exporting USD files
+
+USD files can be created by exporting GameObjects selected in the Hierarchy using the USD context menu options, or using the Recorder window. Supported USD file formats are:
+
+* USDC (.usd or .usdc)
+* USDA (.usda)
+* USDZ (.usdz)
+
+The default file format when exporting is .usd with USDC encoding, but changing the file ending in the save menu to .usda will export as USDA.
+
+## Export Selected with Children
+
+On Mac you will see multiple file format options- USDC and USDA.
+
+On Windows the default is .usd with a USDC format, but you can change the file ending in the Save window to specify a different file ending and encoding.
+
+
+## Export Transform Overrides
+
+Modifications to transforms in USD files can be exported as overrides to the original files. 
+
+Overrides can be exported from the USD menu with the GameObject containing the [UsdAsset](/USD-in-the-Editor#USD-Asset) component selected, which will export transform overrides for the entire hierarchy. Alternatively, you can also export just the overrides when exporting from the Recorder window by changing the 'Override Setting' to 'Export Transform Overrides Only'.
+
+
+For full asset pipeline flexibility these override files do not include a reference to the original file, but this can be added by manually adding a sublayer in the header of the resulting USDA file:
+
+
+```
+#usda 1.0
+(
+    defaultPrim = "myCube"
+    endTimeCode = 0
+    startTimeCode = 0
+    upAxis = "Y"
+    subLayers = [
+        @c:/path/to/original/myCube.usda@
+    ]
+)
+
+
+over "myCube"
+{
+    ...
+}
+```
+
+
+Note: Modifications to the transform of the Root GameObject are not currently reflected in the override, as Unity assumes the root in all USD files is at the origin.
+
+## Export Selected as USDZ
```

Comment:
we should note that Unity GameObjects set with "EditorOnly" tag will be ignored for export